hw/ip/prim/dv/prim_prince/prim_prince_sim.core

```diff
@@ -0,0 +1,28 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:prim_prince_sim:0.1"
+description: "PRINCE block cipher DV sim target"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:prim:all
```

Comment:
Perhaps set this to only the dependencies you need?

Reply:
Currently there are no core files for the standalone scrambler primitives, I can create some and use those instead.

Reply:
Aah I see. Then leave it the way it is.